.eslintrc.json

```diff
@@ -180,7 +180,7 @@
             }
         ],
         "react/jsx-pascal-case": "error",
-        "react/jsx-sort-default-props": [
+        "react/sort-default-props": [
```

Comment:
this rule was renamed: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-sort-default-props.md